extensions/azure/blob/provision/src/test/java/org/eclipse/dataspaceconnector/provision/azure/blob/ObjectContainerStatusCheckerTest.java

```diff
@@ -57,28 +57,19 @@ void isComplete() {
         putBlob("hello.txt", helloTxt);
         putBlob(testRunId + ".complete", helloTxt);
 
-        assertThat(checker.isComplete(createResource(containerName))).isTrue();
+        assertThat(checker.isComplete("123", emptyList())).isTrue();
```

Comment:
"123" is not the right name.
```suggestion
        assertThat(checker.isComplete(bucketName, emptyList())).isTrue();
```

Reply:
Fixed in the patch